.github/workflows/staging.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         key: ${{secrets.STAGING_SSH_PRIVATE_KEY}}
         inventory: |
           [servers]
-          despina.uberspace.de ansible_user=staghios domain=staging.ephios.de smtp_mailbox=noreply admin_email=admin@ephios.de
+          despina.uberspace.de ansible_user=staghios domain=staging.ephios.de smtp_mailbox=noreply admin_email="Ephios Admins <admin@ephios.de>"
```

Comment:
I hope this works? Didn't get emails from staging. Although I still find this format unintuitive.

Reply:
yes this should be correct.
well this seems to be some kind of standard format, at least `from email.utils import getaddresses` is used, so this comes from python directly